pkg/cmd/provisioning/alibabacloud/delete-ram-users.go

```diff
@@ -89,15 +89,42 @@ func deleteComponentUser(client alibabacloud.Client, userName string) error {
 	return err
 }
 
+func getUsersToDelete(client alibabacloud.Client, name, credReqDir string) ([]string, error) {
+	usersToDelete := make([]string, 0)
+	if credReqDir != "" {
+		// Process directory
+		credRequests, err := provisioning.GetListOfCredentialsRequests(credReqDir)
```

Comment:
I thought the purpose of this PR was to not need the `credReqDir` to be provided in the `delete` subcommand?

Reply:
I'd rather just delete the `credReqDir` since finding the resource(s) by using the `--name` parameter should be sufficient for finding the resources that need to be cleaned up. And the `--name` parameter is already required, so we already have all the information needed to do a proper cleanup.

Reply:
ok, have removed the `credReqDir`